docker-for-windows/index.md

```diff
@@ -593,11 +464,9 @@ because they do not apply to Windows containers:
 
 ### Kubernetes
 
-**Kubernetes is only available in Docker for Windows 18.02 CE Edge.** Kubernetes
-support is not included in Docker for Windows 18.02 CE Stable.
-
-To find out more about Stable and Edge channels and how to switch between them,
-see [General configuration](#general).
+[Kubernetes on Docker for Windows](/docker-for-windows/kubernetes/){: target="_blank" class="_"}
+is available in
+[18.02 Edge (win50)](/docker-for-windows/edge-release-notes/#docker-community-edition-18020-ce-rc1-win50-2018-01-26){: target="_blank" class="_"} and higher.
```

Comment:
Our anchors are really bad, we should add simpler links in there and use them.  We could for instance have simply `#win50`.

Reply:
agree